### DISCUSSION TITLE
Want to view or integrate 2D drawings with IFC in viewer

### DISCUSSION THREAD

**Discussion by jahyun-koo** [IfcOpenShell/IfcOpenShell#4540] [Q&A]
Hello,

I'm currently using Revit to export my IFC files and viewing them in a viewer. However, while working in this environment, I'm keen to be able to view the 2D drawings I used in Revit within the IFC viewer at the same time.

I'm curious if there's a way to insert drawings onto a specific floor in the IFC file, or if it's possible to expose a DXF file, which has been exported from Revit, in the viewer. I believe these options could significantly enhance the efficiency and understanding of my project.

If anyone has insights or good advice on how to work in this direction, please share your thoughts. Any help would be greatly appreciated.

Best regards.

**Comment by aothms** [comment]
If you use ifc-pipeline / https://view.ifcopenshell.org/ you have a 2D SVG widget on the side in which you can view generated (!) views from the model. Interoperability on the actual drawings in native authoring tools are minimal. And 2D drawing formats typically don't have meta-data to incorporate them into the 3D view (we do have that kind of metadata on the SVGs we generate). So I think it depends a bit what you are really after and also concretely what viewer you would like to use.
